docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -25,13 +25,12 @@ government agencies.
 | [IA-03 Device Identification and Authentication]((=fedramp.control_url=)IA-3) | Teleport requires valid x509 or SSH certificates issued by a Teleport Certificate Authority (CA) to establish a network connection for device-to-device network connection between Teleport components. |
 | [SC-12 Cryptographic Key Establish and Management]((=fedramp.control_url=)SC-12) | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed).<br/>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. <br/>In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
 
-Enterprise customers can download the custom FIPS package from the [Dashboard](https://dashboard.gravitational.com/web/login).  Look for `Linux 64-bit (FedRAMP/FIPS)`. RPM and DEB packages are also available.
-
 ## Setup
 
-Visit the [Teleport Customer
+Visit your [Teleport Account](https://teleport.sh) or the legacy [Teleport Customer
```

Comment:
We will deprecate `dashboard.gravitational.com` soon. There is no need to keep any references. Any customer who needs new license, can get it from `teleport.sh` instead.

Reply:
done

Reply:
Please take a look when you can @klizhentas